similarity/config/test2.configurations.properties

```diff
@@ -0,0 +1,20 @@
+memorybasedESA	src/main/resources/MemoryBasedESA.txt
+pageIDMapping	src/main/resources/wikiPageIDMapping.txt
+word2vec	src/main/resources/vectors-enwikitext_vivek200.txt
+phrase2vec	/shared/bronte/sling3/data/phraseEmbedding.txt
```

Comment:
this is not portable.  If you need a local path because the resource is large, please instead create a stub resource file with only a few entries relevant to the test and simply add it in src/test/resources (not src/main/resources)